virttest/utils_conn.py

```diff
@@ -1858,19 +1894,25 @@ def conn_setup(self):
         unix_sock_ro_perms = self.unix_sock_ro_perms
         unix_sock_rw_perms = self.unix_sock_rw_perms
         access_drivers = self.access_drivers
+        sasl_type = self.sasl_type
         restart_libvirtd = self.restart_libvirtd
         client_session = self.client_session
+        server_session = self.server_session
+        remote_session = None
+        server_ip = self.server_ip
+        server_user = self.server_user
+        server_pwd = self.server_pwd
         sasl_allowed_username_list = self.sasl_allowed_username_list
         traditional_mode = self.traditional_mode
 
         # edit the /etc/libvirt/libvirtd.conf to add auth_unix_ro arg
-        if auth_unix_ro:
+        if auth_unix_ro != 'none':
```

Comment:
What is the reason for this change? 
If auth_unix_ro == 'none', we should still do the replacement because the default value with '#' in the beginning is polkit, not 'none'.

Reply:
Unfortunately, the dictionary is initialized to 'none' at line 1747 if there is no any other value provided to method. That means, the value of auth_unix_ro was always set to none, when no other value was provided. Same case was for auth_unix_rw and similar to unix_sock_ro_perms, unix_sock_rw_perms. Those changes are not intended by test and therefore when no value is provided to UnixConnection class, the default values in libvirtd.conf or libvirt.conf config files are used.

Reply:
I wonder if we can update the default value of these settings to None or just remove the default values?  I think it might be an issue if we need to set auth_unix_ro to 'none' in some cases... What do you think？ @kamilvarga @dzhengfy

Reply:
@kamilvarga, after discussion with @Yingshun , we think  'none' is a valid value for those parameters, so below default value 'none' should be removed.  ` init_dict['auth_unix_rw'] = init_dict.get('auth_unix_rw', 'none')` 
Then there will not be a 'none' value for those parameters. So your change for L1909` if auth_unix_ro != 'none':` can be removed.

Reply:
Will check.

Reply:
Updated as suggested.